eventq_rabbitmq/lib/eventq_rabbitmq/rabbitmq_queue_manager.rb

```diff
@@ -42,6 +42,11 @@ def get_subscriber_exchange(channel, queue)
         return channel.fanout("#{_queue_name}.ex")
       end
 
+      def get_delay_exchange(channel, queue)
+        _queue_name = EventQ.create_queue_name(queue.name)
+        channel.fanout("#{_queue_name}.d.ex")
```

Comment:
I think you will need to add the delay length to the exchange name:

`channel.fanout("#{_queue_name}.#{delay}.d.ex")`